src/sentry/uptime/models.py

```diff
@@ -57,6 +57,9 @@ class UptimeSubscription(BaseRemoteSubscription, DefaultFieldsModelExisting):
     headers = JSONField(json_dumps=headers_json_encoder, db_default=[])
     # HTTP body to send when performing the check
     body = models.TextField(null=True)
+    # How to sample traces for this monitor. Note that we always send a trace_id, so any errors will
+    # be associated, this just controls the span sampling.
+    trace_sampling = models.BooleanField(default=False)
```

Comment:
You're going to need to add this to the monitor unique constraint as well. It will also need to be part of a query somewhere that gets or creates uptime subscriptions

Reply:
added. and was planning on having this PR be the migration only, if that's okay?